data/assets/examples/screenspacerenderable/screenspaceinsetblackout/screenspaceinsetblackout.asset

```diff
@@ -0,0 +1,45 @@
+-- basic
+-- Create a screenspace image plane with controls to for a blackout shape.
+-- Can be used when using a secondary projector to project content on a dome surface.
+
+local texturePath = asset.resource({
+  Type = "HttpSynchronization",
+  Identifier = "screenspaceinsetblackout_calibration_pattern",
+  Name = "ScreenSpaceInsetBlackout Calibration Pattern",
+  Version = 1
+})
+
+local inset = {
+  Identifier = "ScreenSpaceInsetBlackout_Example",
+  Type = "ScreenSpaceInsetBlackout",
+  Name = "ScreenSpaceInsetBlackout Example - Basic",
+  Blackoutshape = {
+    -- Must always contain four corners in the following order: 
+    -- top-left, top-right, bottom-right, bottom-left
+    Corners = { {0.0, 1.0}, {1.0, 1.0}, {1.0, 0.0}, {0.0, 0.0} },
+    
+    -- Contains control points for top spline in order left to right
+    Top = { },
+    
+    -- Contains control points for right spline in order top to bottom
+    Right = { },
+    
+    -- Contains control points for bottom spline in order right to left
+    Bottom = { },
+    
+    -- Contains control points for left spline in order bottom to top
+    Left = { },
```

Comment:
Are these needed?  Looks like they might be optional?

How about making them optional, and then making two examples: One which has these control points set and one that hasn't?

Reply:
Oh, they already are optional. So then I suggest removing this part from the example and just creating a second example with some control points